src/main/java/stringcalculator/StringCalculator.java

```diff
@@ -0,0 +1,21 @@
+package stringcalculator;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+public class StringCalculator {
+	private final OperandParser parser = new OperandParser();
+
+	private final List<Operand> operands = new ArrayList<>();
+
+	public StringCalculator(final String input) {
+		this.operands.addAll(parser.parse(input));
+	}
+
+	public BigInteger calculate() {
+		return operands.stream()
+			.map(Operand::value)
+			.reduce(BigInteger.ZERO, BigInteger::add);
+	}
```

Comment:
`Operand` [객체에게 메시지를 보내](https://tecoble.techcourse.co.kr/post/2020-04-28-ask-instead-of-getter/)보면 어떨까요? 
```suggestion
	public Operand calculate() {
		return operands.stream()
			.reduce(Operand.ZERO, Operand::add);
	}
```